config/module_params.config

```
@@ -8,7 +8,8 @@ params{
 
 
   // cell type consensus
+  cell_type_blueprint_ref_file = 'https://raw.githubusercontent.com/AlexsLemonade/OpenScPCA-analysis/e4ffb305a87a54133f398c4445ff92348fcb9d8a/analyses/cell-type-consensus/references/blueprint-mapped-ontologies.tsv'
   cell_type_panglao_ref_file = 'https://raw.githubusercontent.com/AlexsLemonade/OpenScPCA-analysis/b870a082bc9acd3536c5f8d2d52550d8fe8a4239/analyses/cell-type-consensus/references/panglao-cell-type-ontologies.tsv'
```

Filling tags in for hashes
```suggestion
  cell_type_blueprint_ref_file = 'https://raw.githubusercontent.com/AlexsLemonade/OpenScPCA-analysis/refs/tags/v0.2.1/analyses/cell-type-consensus/references/blueprint-mapped-ontologies.tsv'
  cell_type_panglao_ref_file = 'https://raw.githubusercontent.com/AlexsLemonade/OpenScPCA-analysis/refs/tags/v0.2.1/analyses/cell-type-consensus/references/panglao-cell-type-ontologies.tsv'
```